pyproject.toml

```
@@ -50,9 +49,7 @@ test = [
     "pytest~=8.0",
     "hypothesis~=6.108",
 ]
-lint = [
-    "yamllint~=1.34",
-]
+lint = []
```

```suggestion
lint = [
    # Left empty for scaffolding and Makefile compatibility
]
```

Out of consistency with starbase